### ISSUE TITLE
genome size when working with human data

### ISSUE THREAD

**Issue by wisekh6** [marbl/canu#124]
Hello,

I wonder what value would be most appropriate when applying canu to PacBio low coverage sequencing from a human sample.
Your comments will be highly appreciated.

Thank you,

HK

**Comment by skoren** [comment]
I'm not sure what you're asking? The genome is the size of the genome which we've generally set to 3.1gb for human assemblies but it doesn't depend on coverage. If you have less than 30X you should use the sensitive parameters
```
corMhapSensitivity=high corMinCoverage=0 errorRate=0.035 
```

**Comment by wisekh6** [comment]
Thank you very much for your quick response.
I am sorry if my question was trivial :).

There was a genomeSize parameter required for canu, and I thought that it should be guessed in a different way if, for example, I would like to work with tumors having lots of insertions:
[hkim6@cnode337 bwamem]$ canu --help
```
-- Detected Java(TM) Runtime Environment '1.8.0_45' (from 'java').
usage: canu [-correct | -trim | -assemble] \
            [-s <assembly-specifications-file>] \
             -p <assembly-prefix> \
             -d <assembly-directory> \
             genomeSize=<number>[g|m|k] \
             errorRate=0.X \
            [other-options] \
            [-pacbio-raw | -pacbio-corrected | -nanopore-raw | -nanopore-corrected] *fastq

```
So, you mean '3.1gb' is genomeSize for human assemblies. 

And, may I ask another question?
Actually, I could create contigs.fasta by applying canu to (low coverage) PacBio subreads fastqs. I used the sensitive parameters as suggested by you. Then, I evaluated the assembled contigs by using 'MUMmer' package with a part of human genome reference (a specific genomic region ~200 kb). I found that ~1/4 of the genomic region was not covered with any contig sequences. My question is: Is it normal? Originally, I expected that most of the regions would be coveraged by the contigs.

Thank you in advance,

HK

**Comment by skoren** [comment]
Genome size doesn't affect very much in the assembly other than the amount of data to be corrected but since you have a low coverage dataset, all the data would be corrected anyway.

If you have >15X you generally will have a complete genome. Mapping to the human reference you would cover 85-90% of the reference with default parameters. However, depending on the region, default mummer parameters may not find hits, especially if it is repetitive so you could have contigs that represent the region but aren't mapped.

**Comment by wisekh6** [comment]
Thank you very much for clarifying it.

May I ask you another question?
Actually, I was interested in only assembling contigs that would be mapped to human chr7.
For this goal, I used 'bwa' in order to select candidate sequencing reads predicted to be mapped to chr7. Only using the filtered reads, I performed canu in order to create assembled contigs. Shown below is the dot plot comparing chr7 reference and the assembled contigs, I wonder if you think my approach only using the filtered reads would be ok.

Thank you very much,

HK

![my_favorite_organism assemblytics dotplot_filtered](https://cloud.githubusercontent.com/assets/3599405/15027786/16c19bb4-120b-11e6-8c3e-867db01dd879.png)
